tests/test_lib/test_asc.py

```diff
@@ -1,232 +0,0 @@
-
-    def test_student_half_active(self, app):
-        """A student who is active on one team is considered an active athlete."""
-        sid = '1234567890'
-        asc_data = [
-            asc_data_row(
-                sid,
-                'WWP',
-                'Women\'s Water Polo',
-                'WWP',
-                'Women\'s Water Polo',
-                '2017-18',
-                'No',
-                'Yes',
-                status='Not Active',
-            ),
-            asc_data_row(
-                sid,
-                'WFH',
-                'Women\'s Field Hockey',
-                'WFH',
-                'Women\'s Field Hockey',
-                '2017-18',
-                'Yes',
-                'Yes',
-                status='Practice',
-            ),
-            asc_data_row(
-                sid,
-                'WTE',
-                'Women\'s Tennis',
-                'WTE',
-                'Women\'s Tennis',
-                '2017-18',
-                'No',
-                'Yes',
-                status='Not Squad',
-            ),
-        ]
-        # Run import script
-        status = asc.merge_student_athletes(asc_data, delete_students=False)
-        assert 1 == status['new_students']
-        saved_student = Student.find_by_sid(sid)
-        assert saved_student.is_active_asc is True
-        assert len(saved_student.athletics) == 1
-        assert saved_student.athletics[0].group_code == 'WFH'
-
-    def test_student_intensive(self, app):
-        """Marks intensive status if set."""
-        jane_sid = '1234567890'
-        polo_code = 'WWP'
-        asc_data = [
-            asc_data_row(
-                jane_sid,
-                polo_code,
-                'Women\'s Water Polo',
-                'WWP',
-                'Women\'s Water Polo',
-                '2017-18',
-                'Yes',
-                'Yes',
-            ),
-        ]
-        # Run import script
-        status = asc.merge_student_athletes(asc_data, delete_students=False)
-        assert 1 == status['new_students']
-        saved_student = Student.find_by_sid(jane_sid)
-        assert saved_student.in_intensive_cohort
-        assert saved_student.is_active_asc is True
-
-    def test_ambiguous_group_name(self, app):
-        sid = '1234567890'
-        ambiguous_group = 'MFB'
-        asc_data = [
-            asc_data_row(
-                sid,
-                ambiguous_group,
-                'Football',
-                ambiguous_group,
-                'Football',
-                '2017-18',
-                'Yes',
-            ),
-        ]
-        # Run import script
-        status = asc.merge_student_athletes(asc_data, delete_students=False)
-        assert 1 == status['new_team_groups']
-        eccentric_football_group = Athletics.query.filter_by(group_code=ambiguous_group).first()
-        assert 'Football - Other' == eccentric_football_group.group_name
```

Comment:
Removal of tests above is no great loss. They are oriented towards 'model' verification and the models are gone.